portfolio/src/main/java/com/google/sps/servlets/Event.java

```diff
@@ -21,12 +23,14 @@ public Event(String organizer, String eventName, String location, String descrip
         this.end = end;
         this.password = generatePassword();
         this.key = key;
+        this.centerCoord = centerCoord;
+        this.pathCoords = pathCoords;
     }
 
     private String generatePassword(){
         String password = "";
         for (int i = 0; i < 5; i++){
-            char rand = (char)((int)(Math.random() * 97) + 33); 
+            char rand = (char)((int)(Math.random() * 93) + 33); 
```

Comment:
why the change from 97 to 93?

Reply:
I noticed some chars that were not usual alphanumeric values, and adjusted the number to correct a previous calculation error.